src/Kmd.Logic.Cvr.Client/CvrClient.cs

```diff
@@ -19,12 +19,13 @@ namespace Kmd.Logic.Cvr.Client
     /// - Create a CVR configuration for the distribution service being used.
     /// </remarks>
     [SuppressMessage("Design", "CA1001:Types that own disposable fields should be disposable", Justification = "HttpClient is not owned by this class.")]
```

Comment:
This one should implement IDisposable (as it has disposable members) - see my PR :)

Reply:
done